andela/ah-frontend-zeus
# Ft bookmark article 162163146

**What does this PR do?**
This Pull Request adds the following functionality to the articles app:
	•user can view his or her profile

**Description of Task to be completed?**
The logged in user can bookmark an article to read later.

**How should this be manually tested?**
- Install dependencies: `yarn`
- Run the application server: `yarn start`
- To see the profile go to `https.zeus-fe.herokuapp.com/articles` in the browser and select any article. 
- click the bookmark icon on the article. 

**What are the relevant pivotal tracker stories?**
[#162163146](https://www.pivotaltracker.com/story/show/162163146)

** screenshots
<img width="994" alt="screen shot 2019-01-31 at 19 10 39" src="https://user-images.githubusercontent.com/22595454/52067643-16391680-258c-11e9-822d-1c3b60a45d76.png">
